tests/api/admin/controller/test_report.py

```diff
@@ -36,15 +39,3 @@ def test_generate_inventory_report(self, report_fixture: ReportControllerFixture
             assert response.status_code == HTTPStatus.ACCEPTED
             body = json.loads(response.data)  # type: ignore
```

Comment:
Minor: you can directly get the json from a response by calling `response.json`.